qa-heal.planx-pla.net/portal/gitops.json

```diff
@@ -385,29 +385,54 @@
     },
     "tagCategories": [
       {
-        "name": "Demographic Info",
-        "color": "rgba(64, 34, 100, 1)",
+        "name": "Study Setting",
+        "color": "#532565",
+        "display": true
+      },
+      {
+        "name": "Subject Characteristics",
+        "displayName": "Subject Characteristics",
+        "color": "#982568",
         "display": true
       },
       {
         "name": "Study Design",
-        "color": "rgba(153, 40, 107, 1)",
+        "displayName": "Study Design",
+        "color": "#981f32",
         "display": true
       },
       {
-        "name": "HEAL Specific",
-        "color": "rgba(161, 36, 46, 1)",
+        "name": "Intervention/Treatment",
+        "displayName": "Intervention/Treatment",
+        "color": "#bf362e",
+        "display": true
+      },
+            {
+        "name": "Pain",
+        "color": "#e07c3e",
```

Comment:
is `displayName` required?

Reply:
No it's going to be "Pain". I had `displayName` for the others to keep both names in capitals because the service returns every second word in low capitals for every tag name with two words.